packages/web/utils/questChains.ts

```diff
@@ -193,6 +231,144 @@ export const QuestChainGreatHousesDetails: Record<
   },
 };
 
+export const QuestChainPlaybooksDetails: Record<
+  QuestChainType,
+  QuestChainInfo
+> = {
+  [QuestChainsPlaybooks.MEME_DRIVEN_ORGANIZATIONS]: {
```

Comment:
Seems like these playbooks should be in a database table... Do you have a good reason why they shouldn't be?  I assume we will want to be modfiying them fairly often, which shouldn't require mucking in the code

Reply:
I can't set up the hasura console on my mac, been at it for hours and then I gave up. Also it's convenient and simpler to change the config if it's in the client.

Reply:
OK. Definitely don't agree with this 

> Also it's convenient and simpler to change the config if it's in the client.

when you can just update the data in Hasura and have it reflect in any environment in a matter of seconds...

Reply:
@dysbulic Any thoughts here? Should we get this stuff in the DB or merge as-is

Reply:
> when you can just update the data in Hasura and have it reflect in any environment in a matter of seconds...

don't you have to write a migration?

Reply:
Anyways, I can't even spin up a hasura console.